### DISCUSSION TITLE
Align View To / Look At Sketch Plane Button

### DISCUSSION THREAD

**Discussion by SupinePandora43** [hlorus/CAD_Sketcher#74] [Feature Requests]
**Motivation / Existing Problem**
You're not able to look at sketch plane with an angle straight from top or bottom.

**Description**
Make a button (also execute it on sketch plane creation) that will rotate camera so that sketch plane will be at 90°

**Alternative Solutions**
Modify viewport presets (that sphere on top right or numpad) to match plane's rotation.

**Mockup**
Set camera's rotation to `sketchplane_normal * -1`

**Comment by hlorus** [comment]
Agree it would be awesome to have that functionality, @mikemac30 actually provided a PR, see:
https://github.com/hlorus/CAD_Sketcher/pull/132.

Lets hope this will land in the addon :)

**Comment by hlorus** [comment]
This is now planned for 1.0, see: https://github.com/hlorus/CAD_Sketcher/issues/131
